src/sh/keptn/Keptn.groovy

```diff
@@ -1,5 +1,9 @@
 package sh.keptn
 
+import org.jenkinsci.plugins.plaincredentials.StringCredentials
+import com.cloudbees.plugins.credentials.CredentialsProvider
```

Comment:
just double checking: are these cloudbees plugins "standard" plugins that come with a default jenkins installation? or do we add a new dependency here?

Reply:
Both dependencies are from plugins that come with the default jenkins installation
* https://plugins.jenkins.io/plain-credentials/
* https://plugins.jenkins.io/credentials/

Still can't hurt to add them to the list of required jenkins plugins for the library